microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,32 +65,51 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 
 1. Under the **Microsoft Edge** program, select **Publish API**.
 
 1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page:
+   The API credentials have now been created; you've enabled or renewed the API.  The following items are now displayed on the **Publish API** page:
+   * **Client ID**
+   * **Client secret** (for REST Publish API v1) or **API key** (for REST Publish API v2)
+   * **Expiry date**
+   * **Access token URL** (for REST Publish API v1)
+
+   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)<!-- todo: update per v2; maybe show a v2 png and then a v1 png -->
 
-   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
+1. Write down the following:
+   * The **Client ID**.
+   * The **Client secret** (REST Publish API v1) or **API key** (REST Publish API v2).
+   * The **Access token URL** (REST Publish API v1).
 
-1. Write down the **Client ID**, **Client secret** and the **Access token URL**.  You'll use these values in the next step, to get an access token.
+   You'll use these values in the next step, to get an access token.
+
+<!-- todo: update steps to mention selecting v2 vs v1.
+Add screenshot png?
+What's the title of the page that contains the UI control to select v1 vs v2?
+What's the label of the UI control?
+Is the UI control an Option button, or a Button?
+-->
 
 > [!IMPORTANT]
-> Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials). This particular secret isn't shown again.
+> Applies to REST Publish API v1, not v2:
+> Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials).  This particular secret isn't shown again.
+
+You can generate multiple client secrets (if using v1) for your client ID.  For example, you can create multiple secrets for multiple projects.
 
-You can generate multiple client secrets for your client ID.  For example, you can create multiple secrets for multiple projects.
+When using REST Publish API v2, the ClientId and API Keys are automatically re-generated, by backend services.<!-- todo: every time you use any endpoint of the REST API?  what action causes re-generation of ClientId and API Keys? -->
 
 
 <!-- ====================================================================== -->
 ## Retrieving the access token
```

Comment:
Retrieving the access token is only for v1 and not v1.1

Reply:
implemented tabsets to completely separate v1.1 info/steps vs. v1 info/steps